lib/graphql/client.ex

```diff
@@ -107,4 +107,34 @@ defmodule BridgeEx.Graphql.Client do
       log_response_on_error: false
     )
   end
+
+  @doc """
+  Formats Graphql query variables to make it compliant with the Schema
+  """
+  @spec format_variables(any) :: any
+  def format_variables(nil), do: nil
+  def format_variables(variable = %Date{}), do: Date.to_string(variable)
+  def format_variables(variable = %DateTime{}), do: DateTime.to_string(variable)
+  def format_variables(variable = %NaiveDateTime{}), do: NaiveDateTime.to_string(variable)
+  def format_variables(variable) when is_boolean(variable), do: variable
+
+  def format_variables(variable) when is_map(variable) do
+    variable
+    |> Enum.map(fn
+      {key, value} when is_atom(key) ->
+        {key |> Atom.to_string() |> Absinthe.Utils.camelize(lower: true), format_variables(value)}
+
+      {key, value} when is_binary(key) ->
+        {Absinthe.Utils.camelize(key, lower: true), format_variables(value)}
+    end)
+    |> Map.new()
+  end
+
+  def format_variables(variable) when is_atom(variable),
+    do: variable |> Atom.to_string() |> String.upcase()
```

Comment:
Even in this case, not all the applications want to convert any atom to upper case.

Currently this function is changing some values, but it should do only what is declared in the doc:
> transforms snake_case variable names in compliant camelCase